src/lib/components/bottom-nav-settings.svelte

```diff
@@ -24,12 +24,17 @@
 </script>
 
 {#if open}
-  <div class="flex h-full flex-col justify-end gap-6 overflow-auto px-4 py-8">
+  <div class="flex h-full flex-col justify-start gap-6 overflow-auto px-4 py-8">
     <TimezoneSelect position="left" />
     <NavigationButton
       onClick={onCodecServerClick}
       tooltip={translate('data-encoder.codec-server')}
       label={translate('data-encoder.codec-server')}
+      data-testid="data-encoder-status{hasCodecServer
+        ? $dataEncoder.hasError
+          ? '-error'
+          : '-configured'
```

Comment:
could have the `-` in the the static part of the string, but nbd

Reply:
We use the testid `data-encoder-status` without the `-` if `hasCodecServer` is false.